presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java

```diff
@@ -168,19 +182,35 @@ private PlanNode rewriteQuantifiedEqualsAllApplyNode(ApplyNode node, QuantifiedC
 
         private PlanNode rewriteQuantifiedOrderableApplyNode(ApplyNode node, QuantifiedComparisonExpression quantifiedComparison, RewriteContext<PlanNode> context)
```

Comment:
Please, add a short description to the commit message explaining what causes the problem and how this fixes it. It's hard to reverse-engineer from the code. See a543af5b7c266b829614180d364b6cb9f492d0fa, a6d036211eb33215537e3aa7cd83a76d06ca5b99 or 2d185a4ca61c113a507ea54b3c23fec8e95e4a2a for some examples.